.github/workflows/scripts/gcp-get-cached-disks.sh

```diff
@@ -36,38 +36,36 @@ find_cached_disk_image() {
         echo "Description: ${disk_description}" >&2
         echo "${disk_name}"  # This is the actual return value when a disk is found
     else
-        echo "No ${git_source} disk found." >&2
+        echo "No ${git_source} disk found with '${disk_search_pattern}' pattern." >&2
     fi
 }
 
 # Check if both $DISK_PREFIX and $DISK_SUFFIX are set, as they are required to find a cached disk image
 if [[ -n "${DISK_PREFIX}" && -n "${DISK_SUFFIX}" ]]; then
     # Find the most suitable cached disk image
-    echo "Finding the most suitable cached disk image..."
+    echo "Finding a ${DISK_PREFIX}-${DISK_SUFFIX} disk image for ${NETWORK}..."
     CACHED_DISK_NAME=""
 
-    # First, try to find a cached disk image from the current commit
-    CACHED_DISK_NAME=$(find_cached_disk_image ".+-${GITHUB_SHA_SHORT}" "commit")
-
-    # If no cached disk image is found
-    if [[ -z "${CACHED_DISK_NAME}" ]]; then
-        # Check if main branch images are preferred
-        if [[ "${PREFER_MAIN_CACHED_STATE}" == "true" ]]; then
-            CACHED_DISK_NAME=$(find_cached_disk_image "main-[0-9a-f]+" "main branch")
-        # Else, try to find one from any branch
-        else
+    # Check if main branch images are preferred
+    if [[ "${PREFER_MAIN_CACHED_STATE}" == "true" ]]; then
```

Comment:
If `PREFER_MAIN_CACHED_STATE` is `true` and `find_cached_disk_image` doesn't find anything, then the script fails. Is this the intended behavior? I can open a quick PR to handle this case since I've already given it a thought.

Reply:
This is what I had in mind. https://github.com/ZcashFoundation/zebra/pull/8936

Reply:
Solved with https://github.com/ZcashFoundation/zebra/pull/8908/commits/50cd029a75d4e8dad88e76b3fc233a173b35e0cd